src/App.js

```diff
@@ -1,25 +1,33 @@
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
+import {Home, User, Photo} from "pages";
+import logo from "./assets/AThousandWords.png";
+import { Logo, StyledNav, StyledUl, StyledLi, StyledLink } from "./app.styles";
 
-function App() {
+export default function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <div>
+        <StyledNav>
+          <Link to="/">
+            <Logo src={logo} alt="logo" />
+          </Link>
+          <StyledUl>
+            <StyledLi>
+              <StyledLink to="/">Home</StyledLink>
+            </StyledLi>
+            <StyledLi>
+              <StyledLink to="/user/:userid">User</StyledLink>
+            </StyledLi>
+          </StyledUl>
+        </StyledNav>
+
+        <Switch>
+          <Route component={(props) => <Home {...props} />} exact path="/"></Route>
```

Comment:
why not ` <Route component={Home} exact path="/" />`? you are not passing anything to Home